static/faq.html

```diff
@@ -291,6 +294,55 @@ <h2 id="default-connections">
                 </li>
             </ul>
 
+            <h2 id="securing-data-at-rest">
+                <a href="#securing-data-at-rest">How does GrapheneOS secure data at rest?</a>
+            </h2>
+            <p>GrapheneOS makes full use of the hardware-bound encryption in modern Android devices.
+            The Titan M, a specially designed hardware security module on Pixel 3 handsets handles
```

Comment:
"HSM" is used below without any introduction, so we should introduce it here. Also, the Titan M is present on the 3a
```suggestion
            The Titan M, a specially designed hardware security module (HSM) on Pixel 3 and 3a handsets, handles
```